GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -120,44 +148,204 @@ namespace gdjs {
-          );
+          // Draw points
+          // Center point of the object, with camera movement
+          const centerPoint = {
+            x: object.getDrawableX() + object.getCenterX() - cameraTopLeftX,
+            y: object.getDrawableY() + object.getCenterY() - cameraTopLeftY,
+          };
+
+          // Draw Center point
+          debugDraw.fill.alpha = 0.3;
+          debugDraw.line.color = 0xffff00;
+          debugDraw.fill.color = 0xffff00;
+          debugDraw.drawCircle(centerPoint.x, centerPoint.y, 3);
+
+          if (showPointsNames) {
+            if (!renderedObjectTextPoints.points['center']) {
+              renderedObjectTextPoints.points['center'] = new PIXI.Text(
+                'Center',
+                {
+                  fill: '#ffff00',
+                  fontSize: 12,
+                }
+              );
+            }
+
+            renderedObjectTextPoints.points['center'].position.set(
+              centerPoint.x,
+              centerPoint.y
+            );
+
+            this._pixiContainer.addChild(
+              renderedObjectTextPoints.points['center']
+            );
+          }
+
+          // Draw Origin point
+          // Origin point of the object, with camera movement
+          // For Sprite objects get the position of the origin point.
+          // For others objects that doesn't have origin point the position of the rendered object is used.
+          /*
+          let originPoint = [
+            object.getDrawableX() - cameraTopLeftX,
+            object.getDrawableY() - cameraTopLeftY,
+          ];
+          if (object instanceof gdjs.SpriteRuntimeObject) {
+            originPoint = object.getPointPosition('origin');
+            originPoint[0] = originPoint[0] - cameraTopLeftX;
+            originPoint[1] = originPoint[1] - cameraTopLeftY;
+          }
+          */
+
+          let originPoint = [object.getX(), object.getY()];
+
+          debugDraw.line.color = 0xff0000;
+          debugDraw.fill.color = 0xff0000;
+          debugDraw.drawCircle(originPoint[0], originPoint[1], 3);
+
+          if (showPointsNames) {
+            if (!renderedObjectTextPoints.points['origin']) {
+              renderedObjectTextPoints.points['origin'] = new PIXI.Text(
+                'Origin',
+                {
+                  fill: '#ff0000',
+                  fontSize: 12,
+                }
+              );
+            }
+            renderedObjectTextPoints.points['origin'].position.set(
+              originPoint[0],
+              originPoint[1]
+            );
+
+            // Mark the text points as rendered
+            renderedObjectTextPoints.wasRendered = true;
+            this._pixiContainer.addChild(
+              renderedObjectTextPoints.points['origin']
+            );
+          }
+
+          // Draw custom point
+          if (showCustomPoints) {
+            // @ts-ignore '_animationFrame' does not exist on type 'RuntimeObject | SpriteRuntimeObject'.
```

Comment:
Guard this with a if `object instanceof gdjs.SpriteRuntimeObject` and TypeScript will understand the type of the object:

```suggestion
          if (showCustomPoints && object instanceof gdjs.SpriteRuntimeObject) {
```